src/test/java/crawlercommons/domains/EffectiveTldFinderPSLTest.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Crawler-Commons
+ * Copyright 2019 Crawler-Commons
```

Comment:
I don't know whether we any policies regarding the year of the copyright notice. Afaik, the notices haven't been changed since added in 2016 (0775bb21).

Reply:
I will revert the year changes,

If we decide to change the copyright year, we will do it in a separate commit dedicated to it (and done yearly)

Thanks again @sebastian-nagel

Reply:
Ok, let's wait for comments or open a separate issue for it. And yes, if we decide to do it it should be done yearly (alternatively, in preparation for a release) and for all source files.